src/lib/components/Ideogram.react.js

```diff
@@ -477,15 +472,23 @@ Ideogram.propTypes = {
     dataDir: PropTypes.string,
 
     /**
-     * This is a work in progess and will hopefully be fixed in future releases.
+     * Organism(s) to show chromosomes for. Supply organism's name as a string (e.g. "human") or
+     * organism's NCBI Taxonomy ID (taxid, e.g. 9606) to display chromosomes from a single organism,
+     * or an array of organisms' names or taxids to display chromosomes from multiple species.
```

Comment:
```suggestion
     * or an array of organisms' names or taxIDs to display chromosomes for all these organisms.
```
It might be obvious that "species" and "organisms" mean the same thing in this context, but let's not introduce confusion. Actually, it seems to me that "species" would be more accurate all the way... I see an organism as an instance of a given species.